Makefile

```diff
@@ -10,7 +10,7 @@ OPENAPI_DOCKER_IMG_VERSION=v0.21.0
 
 # Namespace for the agent images
 DOCKER_OUTPUT_NS   ?= aries-framework-go
-AGENT_REST_IMAGE_NAME   ?= agent-rest
+AGENT_REST_IMAGE_NAME   ?= agent-rest-docker
```

Comment:
the image name should not have a docker suffix. this is implied :).

Reply:
fixed